platformio.ini

```diff
@@ -324,45 +311,22 @@ build_flags     = -DUSE_SCREEN=1
                   -DESP32FEATHERTFT=1
                   -DTOGGLE_BUTTON_1=0
                   -DUSE_HSPI_PORT=1
-
-; ===================
-; Effect Set Types
-;
-; These define which effects are included in different project variants
-
-[effects_minimal_flags]
-build_flags     = -DEFFECTS_MINIMAL=1
-
-[effects_demo_flags]
-build_flags     = -DEFFECTS_DEMO=1
-
-[effects_m5demo_flags]
-build_flags     = -DEFFECTS_M5DEMO=1
-
-[effects_full_flags]
-build_flags     = -DEFFECTS_FULL=1
-
-[effects_fan_flags]
-build_flags     = -DEFFECTS_FAN=1
-
-[effects_spectrum_flags]
-build_flags     = -DEFFECTS_SPECTRUM=1
-
-[effects_mesmerizer_flags]
-build_flags     = -DEFFECTS_FULL=1
+                  -DNUM_INFO_PAGES=1
 
 ; ================================================================
 ; Basic environment section, automatically inherited by all others
 ;
 
 [env]
+extends         = base
```

Comment:
This line isn't necessary, and actually ignored. [env] is automatically inherited by all other env blocks (so [base] extends [env] by definition, not the other way around). It is mildly confusing that this block is not at the top of the platformio.ini file, but maybe this is a good time to fix that.